tinydrive/libraries/java.md

```diff
@@ -35,7 +35,7 @@ $ git clone git@github.com:tinymce/tinydrive-java-spring-starter.git
 
 #### 2. Change the `apiKey` in `tinydrive-java-spring-startersrc/main/resources/application.yml`
 
-You get the api key from the `API Key Manager` {{site.accountpage}} page at http://tiny.cloud.
+You get the API key from the `API Key Manager` {{site.accountpage}} page at http://tiny.cloud.
```

Comment:
```suggestion
The API key can be found in the `API Key Manager` {{site.accountpage}} page at http://tiny.cloud.
```
maybe? and I would have thought there would be a variable for http://tiny.cloud and/or that should be a link?